src/common/Button/tests/Button.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import Button from '../Button';
+
+describe('Button component', () => {
+	it('renders the Button component with label', () => {
+		const testProps = {
+			label: 'Click me',
+			onClick: jest.fn(),
+		};
+
+		const { getByText } = render(<Button {...testProps} />);
```

Comment:
In some project, there are restrictions on usage it in this way from eslint. For me, this way is also fine.
```
render(<Button {...testProps} />);

const buttonElement = screen.getByText(testProps.label);
```